packages/format-currency/src/number-format.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Exposes number format capability
+ *
+ * @copyright Copyright (c) 2013 Kevin van Zonneveld (http://kvz.io) and Contributors (http://phpjs.org/authors).
+ * @license See CREDITS.md
+ * @see https://github.com/kvz/phpjs/blob/ffe1356af23a6f2512c84c954dd4e828e92579fa/functions/strings/number_format.js
+ */
+function toFixedFix( n, prec ) {
+	const k = Math.pow( 10, prec );
+	return '' + ( Math.round( n * k ) / k ).toFixed( prec );
+}
+
+export default function number_format( number, decimals, dec_point, thousands_sep ) {
```

Comment:
Now that we're extracting this separately, would it be a worthy effort to update the function to follow our coding standards and best practices?